src/main/java/WebServer.java

```diff
@@ -27,9 +28,15 @@ public void start(int port) throws IOException {
     private void respond() throws IOException {
         serverCommunicationChannel = serverSocketManager.acceptConnection();
         OutputStream output = serverCommunicationChannel.getOutputStream();
-        if (request().contains(SIMPLE_GET_REQUEST)) {
-            responseBuilder = new ResponseBuilder();
-            responseHandler.respond(output, responseBuilder.okayWithEmptyBody());
+        responseBuilder = new ResponseBuilder();
+        String request = request();
+        switch(request) {
+            case SIMPLE_GET_REQUEST:
```

Comment:
what would it take to be able to respond to HEAD from whatever route?

Reply:
Checking if the request contains `HEAD`? e.g. `if request.contains("HEAD")`

Reply:
👍